n_t/smc.py

```diff
@@ -0,0 +1,56 @@
+"""
+Utilities for working with scm++
+"""
+import logging
+import subprocess
+import tskit
+
+
+def write_smcpp_file(path):
+    """
+    Writes a smcpp input file given a treesequence
+    """
+    ts = tskit.load(path)
+    # write a vcf intermediate input
+    with open(path+".vcf", "w") as vcf_file:
+        ts.write_vcf(vcf_file, 2)
+    # index the vcf
+    cmd = ("bgzip " + path + ".vcf")
+    logging.info("Running:" + cmd)
+    subprocess.run(cmd, shell=True, check=True)
+    vz_file = path + ".vcf.gz"
+    cmd = ("tabix " + vz_file)
+    logging.info("Running:" + cmd)
+    subprocess.run(cmd, shell=True, check=True)
+    # run smc++ for vcf conversion
+    smc_file = path + ".smc.gz"
+    cmd = "smc++ vcf2smc " + vz_file + " " + \
```

Comment:
Be nice to standardise on fstrings for this stuff:
```python
cmd = f"smc++ vcf2smc {vz_file} {smc_file} 1 pop 1:"
```

Reply:
made this change and will use fstrings forever now-- hadn't noticed how much typing that saves me